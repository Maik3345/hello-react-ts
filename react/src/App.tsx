import * as React from "react";
import "../global.css";

class App extends React.Component<any, any> {
  public render() {
    return (
      <div className="App">
        <header className="App-header">
          <img
            src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/2000px-React-icon.svg.png"
            className="App-logo"
            alt="logo"
          />
          <h1 className="App-title">Welcome to React</h1>
        </header>
        <p className="App-intro">
          To get started, edit <code>src/App.tsx</code> and save to reload.
        </p>
      </div>
    );
  }
}

export default App;
