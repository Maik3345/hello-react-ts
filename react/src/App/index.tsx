import classnames from 'classnames'
import React from 'react'
import styles from './App.css'
import { FormattedMessage } from "react-intl"

const App = () => {
  return (
    <div className={classnames(styles.App)}>
     
      <header className={classnames(styles.AppHeader)}>
        <img
          src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/2000px-React-icon.svg.png"
          className={classnames(styles.AppLogo)}
          alt="logo"
        />
        <h1 className={classnames(styles.AppTitle)}><FormattedMessage id="welcome" /></h1>
      </header>
      <p className={classnames(styles.AppIntro)}>
        <FormattedMessage id="getstarted" /> <code>src/App.tsx</code>  <FormattedMessage id="saveTo" />
      </p>
    </div>
  )
}

export default App
