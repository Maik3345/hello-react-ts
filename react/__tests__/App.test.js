/* eslint-env jest */
import React from "react";
import { shallow, mount } from "enzyme";
import App from "../App";
describe("Hello React component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<App classes={{}} />);
  });
  it("should be rendered", () => {
    expect(wrapper).toBeDefined();
  });
  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});