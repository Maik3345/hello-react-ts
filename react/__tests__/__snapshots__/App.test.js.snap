// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel component should match snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <img
        alt="logo"
        class="App-logo"
        src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/2000px-React-icon.svg.png"
      />
      <h1
        class="App-title"
      >
        Welcome to React
      </h1>
    </header>
    <p
      class="App-intro"
    >
      To get started, edit 
      <code>
        src/App.tsx
      </code>
       and save to reload.
    </p>
  </div>
</div>
`;

exports[`Hello React component should match snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/2000px-React-icon.svg.png"
    />
    <h1
      className="App-title"
    >
      Welcome to React
    </h1>
  </header>
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      src/App.tsx
    </code>
     and save to reload.
  </p>
</div>
`;
